Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -1275,6 +1275,7 @@ public void removeAllPolylines(String str) {
         ExecutorManager.runOnMainThread(() -> {
             removeMarker("ic_auto_nav_on_map");
             removeMarker("ny_ic_vehicle_nav_on_map");
+            removeMarker("ny_ic_black_yellow_auto");
```

Comment:
can we send a array of marker names needs to be removed as param to this? add all these as falback and mention deprecated